spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -555,385 +540,264 @@ If the @WebService.targetNamespace annotation is on a service
+|Name of the wsdl:operation matching this method.
+|Name of the Java method
 
-a|
-action
-
-a|
-The action for this operation. For SOAP bindings, this determines the
+|action
+|The action for this operation. For SOAP bindings, this determines the
 value of the soap action.
+|""
 
-a|
-""
-
-a|
-exclude
-
-a|
-Marks a method to NOT be exposed as a web method. Used to stop an
+|exclude
+|Marks a method to NOT be exposed as a web method. Used to stop an
 inherited method from being exposed as part of this web service.
 
 If this element is specified, other elements MUST NOT be specified for
 the @WebMethod.
-
-a|
-False
-
+|False
 |===
 
 This member-value is not allowed on endpoint interfaces.
 
-== 4.2.2 Annotation Type Definition
+==== Annotation Type Definition
 
+[source,java]
+----
 @Retention(value=RetentionPolicy.RUNTIME)
-
-@Target(\{METHOD})
-
-public @interface WebMethod \{
-
-String operationName() default "";
-
-String action() default "" ;
-
-boolean exclude() default false;
-
+@Target({METHOD})
+public @interface WebMethod {
+  String operationName() default "";
+  String action() default "" ;
+  boolean exclude() default false;
 };
+----
 
-*4.2.3 Example +
-Java source:*
+==== Example
+*Java source:*
 
+[source,java]
+----
 @WebService
-
-public class MyWebService \{
-
-@WebMethod(operationName = "echoString", action="urn:EchoString")
-
-public String echo(String input) \{
-
-return input;
-
+public class MyWebService {
+   @WebMethod(operationName = "echoString", action="urn:EchoString")
+   public String echo(String input) {
+      return input;
+   }
 }
+----
 
-}
-
-== Resulting WSDL:
+*Resulting WSDL:*
 
+[source, xml]
+----
 <definitions>
-
-<portType name="MyWebService">
-
-<operation name="echoString"/>
-
-<input message="echoString"/>
-
-<output message="echoStringResponse"/>
-
-</operation>
-
-</portType>
-
-<binding name="PingServiceHttpSoap" type="MyWebService">
-
-<operation name="echoString">
-
-<soap:operation soapAction="urn:EchoString"/>
-
-</operation>
-
-</binding>
-
+   <portType name="MyWebService">
+      <operation name="echoString"/>
+         <input message="echoString"/>
+         <output message="echoStringResponse"/>
+      </operation>
+   </portType>
+   
+   <binding name="PingServiceHttpSoap" type="MyWebService">
+      <operation name="echoString">
+         <soap:operation soapAction="urn:EchoString"/>
+      </operation>
+   </binding>
 </definitions>
+----
 
-*__4.3 Annotation: javax.jws.Oneway +
-__4.3.1 Description*
+=== Annotation: javax.jws.Oneway 
+
+==== Description
 
 Indicates that the given web method has only an input message and no
 output. Typically, a oneway method returns the thread of control to the
 calling application prior to executing the actual business method. A
-JSR-181 processor is REQUIRED to report an error if an
+Jakarta Web Services Metadata processor is REQUIRED to report an error if an
 
 operation marked @Oneway has a return value, declares any checked
```

Comment:
this is the rest of the sentence, no need to have it formatted as new paragraph with an extra empty line separator

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/62fdf397f28a148826367da36bcaef2c30e97c7d